integtest/spec/helper/dest.rb

```diff
@@ -9,11 +9,17 @@
 class Dest
   include Sh
 
+  ##
+  # Stdout and stderr of running the conversions in convert_single
+  # or convert_all.
+  attr_reader :convert_outputs
```

Comment:
This allows me to access the output from the conversions without using the member variables that are oh-so-funky to use with rspec. Rather, we end up forwarding all access through a single member variable read that we have tight control over.